Validation: Added Missing Descriptions (#240)
MicrosoftDocs/bingmaps-docs: BingMaps/sdk-native/map-services-api/maplocationaddress-class.md

---
title: MapLocationAddress Class | Microsoft Docs

> description: Describes the MapLocationAddress class for Android and iOS and provides the class's properties and additional references.

Reviewer: ```suggestion
description: Describes the MapLocationAddress class for Android and iOS with descriptions for each of its properties.
```
